ci-operator/step-registry/openshift-qe/write-perfscale-results/openshift-qe-write-perfscale-results-commands.sh

```diff
@@ -1,13 +1,10 @@
 #!/bin/bash
-set -o errexit
-set -o nounset
-set -o pipefail
-set -x
+set +e
 cat /etc/os-release
 oc config view
 oc projects
 python --version
-pushd /tmp
+pushd /tmp  || exit 0
```

Comment:
@paigerube14 Curious about this || exit 0. Is pushd failure not controlled by set +e?

Reply:
One of the automated shell script checks failed without this